### PR TITLE
Fix docs example

### PR DESCRIPTION
**Pull request by Cruiser13**
Did not work as described, some paths changed

docs only

### CODE REVIEW

File: docs/03_Development/01_Extending_Guide/04_Extending_Rule_Actions.md [hunk 1]
```diff
@@ -59,7 +63,7 @@ final class CustomActionType extends AbstractType
 With configuration, comes a Javascript file as well:
 
 ```javascript
-//AppBundle/Resources/public/pimcore/js/custom_action.js
+//web/bundles/app/pimcore/js/custom_action.js
```
Comment (dpfaffenbauer) [inline docs/03_Development/01_Extending_Guide/04_Extending_Rule_Actions.md]: thats not right, the file needs to sit inside the AppBundle and will be symlinked/copied via ```bin/console assets:install```

**Comment by Cruiser13** [comment]
Thanks, fixed.

File: docs/03_Development/01_Extending_Guide/04_Extending_Rule_Actions.md [hunk 2]
```diff
@@ -92,6 +96,7 @@ coreshop.product.pricerule.actions.custom = Class.create(coreshop.rules.actions.
 });
 
 ```
+Don't forget to run bin/console assets:install afterwards to deploy it.
```
Comment (dpfaffenbauer) [inline docs/03_Development/01_Extending_Guide/04_Extending_Rule_Actions.md]: could you please wrap the command inside ``` these characters?

**Comment by dpfaffenbauer** [comment]
alright, I'll merge that now. But for the future, you can inline ````  that as well, so it would look like this ```bin/console ....```

**Comment by dpfaffenbauer** [comment]
thanks :)

**Comment by Cruiser13** [comment]
Thanks @dpfaffenbauer  - the inline variant looked weird in the github preview so I thought it's better this way. But I'll inline it next time.

**Pull request merged**